src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ActiveConfiguredProjectsProvider.cs

```diff
@@ -121,7 +121,7 @@ public async Task<ActiveConfiguredObjects<ProjectConfiguration>> GetActiveProjec
 
             IImmutableSet<ProjectConfiguration> configurations = await _services.ProjectConfigurationsService.GetKnownProjectConfigurationsAsync();
 
-            ImmutableArray<ProjectConfiguration>.Builder builder = ImmutableArray.CreateBuilder<ProjectConfiguration>(configurations.Count);
+            var builder = PooledArray<ProjectConfiguration>.GetInstance();
```

Comment:
Ditto.

Reply:
This was should be reverted, right? We know the size upfront.

Reply:
Strike that, this is only the "maximum", the resulting size might be different, carry one.